src/main/java/com/checkout/HttpMetadata.java

```diff
@@ -0,0 +1,16 @@
+package com.checkout;
+
+import lombok.Data;
+
+import java.util.Map;
+
+@Data
+public class HttpMetadata {
```

Comment:
final

Reply:
This can't be final as all classes extends this one